data/brands/tourism/hotel.json

```diff
@@ -1078,18 +1085,6 @@
         "tourism": "hotel"
       }
     },
-    {
-      "displayName": "Holiday Inn Express & Suites",
```

Comment:
Same with this one. These are different brands and we want to be able to tag them in OSM

Reply:
Based on the research that I've done, it's proven that they're the same brand.

Reply:
They use different signage and they mean different things. If someone wants a suite at their hotel they want to make sure they find the hotel that has suites.

Reply:
I made sure to keep the different signage with my changes, especially since even if there is different signage, they are under the same chain & brand.